recording.py

```diff
@@ -39,7 +48,7 @@ def load_params():
     return params
 
 
-def run_session(params):
+def run_session(params, online, classifier):
```

Comment:
I don't like the idea of mixing online with offline recording, can we please create a different function `run_online_session`?